src/core/Wyam.Configuration/ConfigScript/CacheManager.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+
+using Wyam.Common.Execution;
+using Wyam.Common.IO;
+
+namespace Wyam.Configuration.ConfigScript
+{
+    public class CacheManager
```

Comment:
I like putting all the caching logic in a separate class, nice refactoring. Given that `CacheManager` is only instantiated in a method scope and isn't stored as a member anywhere, does it make sense to mark it `internal`?

Reply:
Sounds good.